CRM/Contribute/BAO/Contribution.php

```diff
@@ -5457,4 +5457,28 @@ public static function createProportionalFinancialEntries($entityParams, $lineIt
     }
   }
 
+  /**
+   * Calculate tax for individual line items.
+   *
+   * @param array $lineItem
+   *   an array of lineitems
+   *
+   * @param int $contributionId
+   *   the id of the contribution
+   *
+   * @return array
```

Comment:
This is incorrect. Could you indicate the data type and precision of the returned contribution total tax amount.